db/004-fallbackphone.sql

```diff
@@ -0,0 +1,22 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 4;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        -- ADD SCRIPT HERE
+        ALTER TABLE locations ADD COLUMN fallback_number text default '+17786810411'; 
```

Comment:
Great point, and something I didn't really think about. In this case, I'm fine w/ the risk since I already get a spam call or two a day, and my phone can be the default fallback until we change it...